script_library/run-ansible.sh

```diff
@@ -29,21 +33,18 @@ function run_ansible(){
     fi
 
     if [[ -f ${extravarsfile} ]]; then
-        echo "Extra variables file exists: $(realpath ${extravarsfile}). Loading its vars in ansible-playbook call."
-        extra_vars="-e @${extravarsfile}"
+        echo "Extra variables file exists: $(realpath "${extravarsfile}"). Loading its vars in ansible-playbook call."
+        ansible_command="$ansible_command -e ${extravarsfile}"
```

Comment:
the @ has disappeared. It needs to stay.

Reply:
ah yes, this failed for me when testing, Im not sure why so I had to remove it

Good catch!

Fake edit: YEs, I remember, it failed because I had the path but there was nothing in it or something like that so it was only adding the `@` at the command line which made it fail.

Reply:
done